spec/features/step_definitions/connectivity_steps.rb

```diff
@@ -2,12 +2,12 @@
 
 step 'I am consuming' do
   set_event_routing
-  initialize_queue
+  set_event_definitions
+  RabbitFeed::ConsumerConnection.instance # Bind the queue
```

Comment:
maybe not that important... but is a bit strange for me that getting the instance has some unexpected side effect that needs to be added here in a comment.. I think creating a `ConsumerConnection.bind_to_queue` which creates the instance might be a bit better?